src/server/routes/api/document/index.js

```diff
@@ -1734,6 +1762,75 @@ const postDoc = provided => {
     } );
 };
 
+/**
+ * @swagger
+ *
+ * /api/document/from-url:
+ *   post:
+ *     description: Create new documents from given Biopax url
+ *     summary: Create new documents from given Biopax url
+ *     tags:
+ *       - Document
+ *     requestBody:
+ *       description: Data to create new Documents
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               url:
+ *                 type: string
+ *     responses:
+ *     '200':
+ *       description: Success
+ *     '500':
+ *       description: Error
+ */
+http.post('/from-url', function( req, res, next ){
+
+  const provided = _.assign( {}, req.body );
+
+  const { url } = provided;
+
+  tryPromise( () => getJsonFromBiopaxUrl( url ) )
+    .then( response => response.json() )
+    .then( docsJSON => new Promise( () => {
+      let pmids = Object.keys( docsJSON );
+      pmids = pmids.filter( pmid => pmid != null );
+      pmids = _.slice( pmids, 0, 2 );
+      // TODO: which email address?
+      let authorEmail = 'pc@biofactoid.com';
```

Comment:
Maybe use an env var here.  That way we can decide on the email address later without code changes

Reply:
@maxkfranz okay I set it through an env variable